.editorconfig

```diff
@@ -1,564 +1,271 @@
-# Version: 4.1.1 (Using https://semver.org/)
```

Comment:
Can you revert your changes to this file and edit it manually? Visual Studio has completely mangled it.

Reply:
yeah, I realize it. going to do that.

Reply:
done.